apollo-federation/src/connectors/models.rs

```diff
@@ -46,6 +46,8 @@ use crate::error::FederationError;
 use crate::error::SingleFederationError;
 use crate::internal_error;
 
+pub(crate) const FRAGMENT_INLINE: &str = "...$fragment";
```

Comment:
```suggestion
pub(crate) const FRAGMENT_INLINE: &str = "...$fragments";
```
I think making this plural communicates a little better that it's a namespace providing multiple fragments.